src/main/kotlin/org/rust/lang/core/completion/CompletionEngine.kt

```diff
@@ -72,6 +73,22 @@ object CompletionEngine {
             ?.mapNotNull { it.libTarget }
             ?.map { LookupElementBuilder.create(extCrate, it.normName).withIcon(extCrate.getIcon(0)) }
             ?.toTypedArray() ?: emptyArray()
+
+    fun completeMod(mod: RsModDeclItem): Array<out LookupElement> {
+        val directory = mod.containingMod.ownedDirectory ?: return EMPTY_ARRAY
+
+        val modFromRsFile = directory.files
+            .filter { file -> file.name.endsWith(".rs") }
```

Comment:
It's more idiomatic to use an implicit `it` argument for such short lambdas: `.filter { it.name.endsWith(".rs") }`